### PR TITLE
deprecation of ellipsometry reader in favor of its own pynxtools-ellips plugin

### PR DESCRIPTION
**Pull request by RonHildebrandt**
Removed the files related to the ellips reader from examples and readme files etc.

This is done to prepare setting up an individual github repository for the pynxtools-ellips plugin.

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-This tool offers parsers/readers/data extractors for various experimental techniques, including
-electron microscopy, photo-emission spectroscopy, optical spectroscopy, atom probe, and other
-techniques. Please refer to the individual README's in each sub-directory for details.
+This tool offers parsers/readers/data extractors for various experimental techniques via
+technique specific plugins.
 
 The examples contain code snippets for creating a NeXus/HDF5 file for the experimental technique
-according to a standardized application definition (e.g. NXem, NXmpes, NXellipsometry, NXapm).
+according to a standardized NeXus application definition (e.g. NXem, NXmpes, NXellipsometry,
+NXapm, NXopt, NXmpes_xps, NXraman).
```
Comment (lukaspie) [inline examples/README.md:10]: it's called NXxps

File: pynxtools/dataconverter/README.md [hunk 2]
```diff
@@ -27,8 +27,7 @@ Options:
                                   arguments instead. The path to the input
                                   data file to read. (Repeat for more than one
                                   file.)
-  --reader [apm|ellips|em_nion|em_om|em_spctrscpy|example|hall|json_map|json_yml|mpes|sts|transmission|xps|xrd]
-                                  The reader to use. default="example"
+  --reader [example]              The reader to use. default="example"
```
Comment (lukaspie) [inline pynxtools/dataconverter/README.md:30]: Is this the new output of `dataconverter --help`?

**Review by mkuehbach — APPROVED** [review]
LGTM

File: pynxtools/dataconverter/README.md (hunk 2 above)
Comment (RonHildebrandt) [reply pynxtools/dataconverter/README.md:30]: no its not, but has to be changed as well in the data converter.

Comment (mkuehbach) [reply pynxtools/dataconverter/README.md:30]: https://github.com/FAIRmat-NFDI/pynxtools/issues/325

**Pull request merged**